Source/Core/DolphinQt/Debugger/CodeViewWidget.h

```diff
@@ -38,6 +44,10 @@ class CodeViewWidget : public QTableWidget
   void ToggleBreakpoint();
   void AddBreakpoint();
 
+  u32 AddressForRow(int row) const;
+
+  std::vector<CodeViewBranch> m_branches;
```

Comment:
This should likely remain private, and instead make `BranchDisplayDelegate` a friend of this class.

This would also allow for safely forward declaring `CodeViewBranch` as well, allowing the definition to be completely hidden.